<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserFeedback::Analyzer::AggregationEditWidget</class>
 <widget class="QWidget" name="UserFeedback::Analyzer::AggregationEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTreeView" name="aggregationView">
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="uniformRowHeights">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionAddAggregation">
   <property name="icon">
    <iconset theme="list-add"/>
   </property>
   <property name="text">
    <string>Add &amp;Aggregation</string>
   </property>
   <property name="toolTip">
    <string>Add a new aggreagation.</string>
   </property>
  </action>
  <action name="actionDeleteAggregation">
   <property name="icon">
    <iconset theme="edit-delete"/>
   </property>
   <property name="text">
    <string>&amp;Delete Aggregation</string>
   </property>
   <property name="toolTip">
    <string>Delete the currently selected aggregation.</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
